### ISSUE TITLE
`BlazeAndroidSyncBuildFlagsProvider` doesn't correctly parse existing `fat_apk_cpu` flag in bazelrc file

### ISSUE THREAD

**Issue by ThomasCJY** [bazelbuild/intellij#3703]
**Background**
After applying the latest bazel android studio plugin. We noticed that our IDE build become much slower and we saw 
```
Forcing fat_apk_cpu flag to a single cpu architecture (x86_64) for sync. Refer to go/auto-set-fat-apk-cpu-to-single-cpu for more details.
```
in our build logs. The regression comes from [this line](https://github.com/bazelbuild/intellij/commit/ad4aa276b54a7f12553c99423f2a57f864638231#diff-4bbe37e21cc3121d4616fec9d8c9c254c6f3696bb2dbb46952ecb6b6f6396e45R55). It turns out `syncOnlyFlags` cannot correctly parse existing `fat_apk_cpu` that's defined in bazelrc. And in our bazelrc, we defined something like 
```
build --config 64bit
build:32bit --fat_apk_cpu=armeabi-v7a
build:64bit --fat_apk_cpu=arm64-v8a
```
but that didn't get picked up in BlazeAndroidSyncBuildFlagsProvider.

**Expected Behavior**
force fat_apk_cpu flag experiment should be set as `off` when there is `fat_apk_cpu` defined in bazelrc

**Comment by kkpattern** [comment]
We also encounter this problem. We currently workaround this by adding following config to project view.

```
sync_flags:
  --fat_apk_cpu=arm64-v8a
```

**Comment by TylerCarberry** [comment]
I also worked around this by adding

```
sync_flags:
  --fat_apk_cpu=armeabi-v7a
```

to my `.aswb/.bazelproject` file

**Comment by sgowroji** [comment]
Thanks for sharing the above workarounds.  Did you get a chance to try the above Shared workarounds. Are you still looking resolution on this request.

**Comment by nkoroste** [comment]
The workaround works but I feel like we need a more official solution for this in order to enable toggling it via bazel flag

**Comment by sgowroji** [comment]
@ThomasCJY can you  disable the experiment by adding the following line to a file `.intellij-experiments` in their home directory

`
blaze.sync.flags.enableForceFatApkCpuExperiment=0`

Separately, as Blaze has deprecated the fat_apk_cpu flag (will this be happening to Bazel as well?), and so this feature will be wholly removed from the plugin in a few releases.

**Comment by mai93** [comment]
BlazeAndroidSyncBuildFlagsProvider is removed in https://github.com/bazelbuild/intellij/commit/583537c63866eac13169ce128bb53aa0017bc6e0, this commit will be included in 10.01.2023 release, please reopen if this did not resolve the issue.
